machine_manager/README.rst

```diff
@@ -0,0 +1,27 @@
+Machine Manager
```

Comment:
You are welcome @bealdav, and I am agree with you that use a common base module and inherit it to extends for the different equipment will be better towards a more generic use, but this means to define a new common module with these generic behaviours and fields.